__mocks__/templates.json

```diff
@@ -26,47 +26,5 @@
     "author": "Platformatic",
     "homepage": "https://example.com",
     "public": true
```

Comment:
Why do we still have this mock @ivan-tymoshenko @tonysnowboardunderthebridge ?

Reply:
Do you mean the "public" property or the mock whole mock file?